Gemfile.lock

```diff
@@ -137,7 +137,6 @@ PATH
 
 GEM
   remote: https://rubygems.org/
-  remote: https://enterprise.contribsys.com/
```

Comment:
we can't check in gemfile changes that remove the sidekick enterprise information

Reply:
ah shoot, my bad. Forgot to make sure this wasn't included in the PR. I'll update.